oqs/oqs.py

```diff
@@ -25,9 +25,10 @@ def _load_shared_obj(name):
     # search typical locations
     paths += [ctu.find_library("oqs")]
     dll = ct.windll if platform.system() == "Windows" else ct.cdll
+    print(paths)
```

Comment:
Is this print statement a debugging statement for you or something that's intended to stay?